Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "nari_act_rust"
```

Comment:
Does this actually need the suffix? Same deal in the lib section.

Reply:
Depends on how we want the import statement to be written.